src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.cpp

```diff
@@ -36,7 +36,11 @@ SelfForceOptions::SelfForceOptions(const double mass_in,
     : mass(mass_in),
       iterations(iterations_in),
       turn_on_time(turn_on_time_in),
-      turn_on_interval(turn_on_interval_in) {}
+      turn_on_interval(turn_on_interval_in) {
+  if (turn_on_interval_in < 1.) {
+    ERROR("The turn on interval must be at least 1 M.");
+  }
```

Comment:
The `lower_bound()` in the option in the `hpp` already takes care of this. No need to check here.